src/textual/widgets/_toggle_button.py

```diff
@@ -218,19 +218,24 @@ def on_click(self) -> None:
     class Changed(Message, bubble=True):
         """Posted when the value of the toggle button changes."""
 
-        def __init__(self, sender: ToggleButton, value: bool) -> None:
+        def __init__(self, toggle_button: ToggleButton, value: bool) -> None:
             """Initialise the message.
 
             Args:
-                sender: The toggle button sending the message.
+                toggle_button: The toggle button sending the message.
                 value: The value of the toggle button.
             """
-            super().__init__(sender)
-            self.input = sender
+            super().__init__()
+            self._toggle_button = toggle_button
             """A reference to the toggle button that was changed."""
             self.value = value
             """The value of the toggle button after the change."""
 
+        @property
+        def control(self) -> ToggleButton:
+            """Alias for self.toggle_button."""
```

Comment:
This will need a reword given that `toggle_button` was moved to being "private". Actually, I'm wondering if `control` should move to the two individual child message classes.